Python/clinic/bltinmodule.c.h

```diff
@@ -247,6 +247,9 @@ builtin_chr(PyObject *module, PyObject *arg)
 
     i = _PyLong_AsInt(arg);
     if (i == -1 && PyErr_Occurred()) {
+        if (PyErr_ExceptionMatches(PyExc_OverflowError)) {
```

Comment:
We shouldn't change the generated Argument Clinic files directly. I'm not sure, unfortunately, if there's an easy way to implement the desired behavior while still using Argument Clinic.